src/components/Chart.jsx

```diff
@@ -1,13 +1,44 @@
-import React from "react";
+import React, { useState } from "react";
 import Card from "./Card";
+import Dropdown from "../components/Dropdown";
 
 const Chart = () => {
+  const [sortBy, setSortBy] = useState("sort by");
+
+  const sortByList = [
+    { code: "Weekly" },
+    { code: "Monthly" },
+    { code: "Quarterly" },
+    { code: "Annually" },
+  ];
+
+  const handleListChange = (input, setter) => {
+    event.preventDefault();
+    setter(input);
+  };
+
   return (
     <Card fullWidth={true}>
-      <h3>Expenses by category</h3>
-      <div className="rounded-xl w-full bg-white dark:bg-gray-900 py-20 mt-6 shadow-sm dark:shadow-md border-2 border-gray-100 dark:border-gray-900"></div>
-      <h3 className="pt-4">Monthly overview</h3>
-      <div className="rounded-xl w-full bg-white dark:bg-gray-900 py-20 mt-6 shadow-sm dark:shadow-md border-2 border-gray-100 dark:border-gray-900"></div>
+      <div className="flex justify-between items-center border-b border-gray-500 pt-6 pl-6 pb-2">
+        <h3 className="text-gray-500 dark:text-gray-400 text-lg font-medium">
+          Expenses by category
+        </h3>
+        <h3 className="text-gray-500 dark:text-gray-400 text-lg font-medium">
+          Expenses overview
+        </h3>
+        <div className="justify-end flex w-28">
+          <div className=" w-full">
+            <Dropdown
```

Comment:
Instead of handling this component in this way, I would rather have it take in a `value` prop and a `onChange` prop. Then in the component, when the dropdown is changed it should simply call the prop `onChange`. This will cause the parent of the dropdown state to update, triggering a change in the value.

Comment here 👇 if you don't understand or need more explination

Reply:
Ah okay I see! Thanks for the feedback 👍👍